examples/reference/streams/bokeh/Tap.ipynb

```diff
@@ -20,13 +20,86 @@
    "outputs": [],
    "source": [
     "import pandas as pd\n",
+    "import panel as pn\n",
     "import numpy as np\n",
     "import holoviews as hv\n",
     "from holoviews import opts\n",
     "\n",
     "hv.extension('bokeh', width=90)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let's look at an extremely simple example. \n",
+    "\n",
+    "We will create an empty `hv.Points` element and set it as the source for the `Tap` stream. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an empty Points element\n",
+    "points = hv.Points([])\n",
+    "# Create the Tap stream with the points element as the source\n",
+    "# We set the x and y here with starting values\n",
+    "stream = hv.streams.Tap(source=points, x=np.nan, y=np.nan)\n",
+    "\n",
+    "# Create a callback for a dynamic map\n",
+    "def location(x, y):\n",
+    "    \"\"\"Create an empty plot with a changing label\"\"\"\n",
+    "    return hv.Points([], label=f'Click at {x:.2f}, {y:.2f}')\n",
+    "\n",
+    "\n",
+    "# Connect the Tap stream to the tap_histogram callback\n",
+    "tap_dmap = hv.DynamicMap(location, streams=[stream])\n",
+    "\n",
+    "# Overlay the Points element (which is linked to the tap stream) with the location plot\n",
+    "points * tap_dmap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's see what it looks like if we used Panel to give us more control over layout and event triggering"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an empty Points element\n",
+    "points = hv.Points([])\n",
+    "# Create the Tap stream with the points element as the source\n",
+    "# We set the x and y here with starting values\n",
+    "stream = hv.streams.Tap(source=points, x=np.nan, y=np.nan)\n",
+    "\n",
+    "# we use panel depends to callback the location function every time the x and y values on the stream change.\n",
+    "@pn.depends(stream.param.x, stream.param.y)\n",
+    "def location(x, y):\n",
+    "    \"\"\"Display pane showing the x and y values\"\"\"\n",
+    "    return pn.pane.Str(f'Click at {x:.2f}, {y:.2f}', width=200)\n",
+    "\n",
+    "# create a container for the plot and the text display\n",
+    "layout = pn.Row(points, location)\n",
```

Comment:
It's up to @philippjfr , but my personal vote is to encourage `pn.bind` rather than `@pn.depends`:

```suggestion
    "# make a function that displays the location when called.\n",
    "def location(x, y):\n",
    "    \"\"\"Display pane showing the x and y values\"\"\"\n",
    "    return pn.pane.Str(f'Click at {x:.2f}, {y:.2f}', width=200)\n",
    "\n",
    "# Display the points and the function output, updated\n",
    "# whenever the stream values change\n",
    "layout = pn.Row(points, pn.bind(location, x=stream.param.x, y=stream.param.y)\n",
```

Reply:
I have never heard of `pn.bind`. Interesting... Are there docs for that?

Reply:
It's recent and sadly not documented. Had to edit the suggestion because it was missing a closing `)`.

Reply:
I agree: it would be better to use `pn.bind` though really here `param.bind` seems more appropriate to me (same thing but doesn't support panel widgets, which aren't being used in this example).

Reply:
param.bind does not yet exist.

Reply:
Right. We should get it into a param dev release at least.